server/http_handler.go

```diff
@@ -1362,3 +1367,45 @@ func (h allServerInfoHandler) ServeHTTP(w http.ResponseWriter, req *http.Request
 	}
 	writeData(w, clusterInfo)
 }
+
+// DBTableInfo is use ro report the db, table info and the current schema version.
+type DBTableInfo struct {
+	DBInfo        *model.DBInfo    `json:"db_info"`
+	TableInfo     *model.TableInfo `json:"table_info"`
+	SchemaVersion int64            `json:"schema_version"`
+}
+
+//ServeHTTP handles request of db info and table info that related to the  tableID.
+func (h dbTableHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	params := mux.Vars(req)
+	tableID := params[pTableID]
+	tblID, err := strconv.Atoi(tableID)
+	if err != nil {
+		writeError(w, errors.Errorf("Wrong tableID: %v", tableID))
+		return
+	}
+
+	schema, err := h.schema()
+	if err != nil {
+		writeError(w, err)
+		return
+	}
+
+	dbTblInfo := DBTableInfo{
+		SchemaVersion: schema.SchemaMetaVersion(),
+	}
+	tbl, ok := schema.TableByID(int64(tblID))
+	if !ok {
```

Comment:
I imagine that this `!ok` means table with ID `tblID` doesn't exist, right? If so, how about returning an error like "Table which ID = %s does not exist", just as how `schemaHandler` did for a non-existing table ID?

I think it doesn't make much sense that user gets a json only containing a schema version for a non-existing table when calling this API.